### PR TITLE
salt: install and configure `crictl` with `kubelet`

### PR DESCRIPTION
**Pull request by NicolasT**
Fixes: #588

### CODE REVIEW

File: salt/metalk8s/kubelet/installed.sls [hunk 1]
```diff
@@ -11,6 +11,22 @@ Install kubelet dependencies:
...
       - conntrack-tools
 
+Install and configure cri-tools:
+  pkg.installed:
+    - name: cri-tools
+    - fromrepo: {{ repo.kubernetes.name }}
+    - require:
+      - pkgrepo: Configure Kubernetes repository
+  file.serialize:
+    - name: /etc/crictl.yaml
```
Comment (ramanduh) [inline salt/metalk8s/kubelet/installed.sls:21]: Maybe we can add the merge_if_exists arg so that any custom option in crictl.yaml would not be overwritten

Comment (ballot-scality) [reply salt/metalk8s/kubelet/installed.sls:21]: Good point

Comment (NicolasT) [reply salt/metalk8s/kubelet/installed.sls:21]: Done

**Comment by NicolasT** [comment]
@bert-e approve

**Review by slaperche-scality — APPROVED** [review]

**Pull request merged**